frontend/public/index.html

```diff
@@ -29,12 +29,6 @@
     <!-- initialize window.prerenderReady to false and then set to true in React app when app is ready for indexing -->
     <script> window.prerenderReady = false; </script>
 
-    <!-- load via script to enable web workers -->
-    <script
-      src="%PUBLIC_URL%/vendor/viz/viz-1.8.0.min.js"
-      type="javascript/worker"
-    ></script>
-
```

Comment:
I wouldn't think we need this since we install `viz.js` as a transitive dependency of `d3-graphviz`, but let me know if I'm missing something

Reply:
This makes me nervous because of that comment above: "load via script to enable web workers". Presumably, if we load `viz.js` as part of our bundle (as opposed to inside a worker `<script>` tag), we disable its webworker functionality.

I'm not familiar enough with `viz` to know if this is ok or not. There's some more details in its readme here: https://github.com/magjac/d3-graphviz#defining-the-hpcc-jswasm-script-tag

But I think we should better understand the impact of this change before making it. Do you mind pulling this viz change out of the PR, and we can address it in a separate PR (if we decided to address it at all)? (And if we decide to leave things as is, we should definitely update that "load via script" comment to include info about _why_ we care about loading it into a webworker).